test/config.sh

```diff
@@ -182,5 +198,5 @@ end_suite() {
     whitely assert_end
 }
 
-WEAVE=../weave
-DOCKER_NS=./bin/docker-ns
+WEAVE=$DIR/../weave
+DOCKER_NS=$DIR/../bin/docker-ns
```

Comment:
140 is using the `run_on` for the weave installed onto `$HOST1`, whereas `weave_on` and 660 are using the local weave script (hence `../weave`).